docs/source/micromamba-installation.rst

```diff
@@ -0,0 +1,202 @@
+
+.. _umamba-install:
+
+=======================
+Micromamba Installation
+=======================
+
+``micromamba`` is a fully statically-linked, self-contained, executable.
+This means that the ``base`` environment is completely empty.
+The configuration for ``micromamba`` is slighly different, namely all environments and cache will be
+created by default under the ``MAMBA_ROOT_PREFIX`` environment variable.
+There is also no pre-configured ``.condarc``/``.mambarc`` shipped with micromamba
+(they are however still read if present).
+
+.. _umamba-install-automatic-installation:
+
+Operating System package managers
+*********************************
+Homebrew
+^^^^^^^^
+
+On macOS, you can install ``micromamba`` from `Homebrew <https://brew.sh/>`_:
+
+.. code:: bash
+
+   brew install micromamba
+
+
+Mamba-org releases
+******************
+Install script
+^^^^^^^^^^^^^^
+
+If you are using macOS or Linux, there is a simple way of installing ``micromamba``.
+Simply execute the installation script in your preferred shell.
+
+For Linux and macOS install with:
+
+.. We use ``bash <(curl ...)`` instead of ``curl .. | bash`` as the latter does not work with prompts
+
+.. code:: bash
+
+   "${SHELL}" <(curl -L micro.mamba.pm/install.sh)
+
```

Comment:
Should we also add the `install.bat` and `install.ps1` scripts on micro.mamba.pm ?

Reply:
I don't know those. Do they exist? Can you add the correct invocation to this PR?